### ISSUE TITLE
Separate update interval setting for each database path

### ISSUE THREAD

**Issue by brezanac** [cboxdoerfer/fsearch#532]
Would it be possible to have a separate update interval setting for each database path?

Here is a use case explaining the reasoning behind the feature request.

I have FSearch configured to update on every start.

One of the paths is targeting an archive which is located on a mechanical HDD drive which is pretty much updated once every year.

![fsearch](https://github.com/cboxdoerfer/fsearch/assets/389770/ac1e0525-c127-41d4-b103-2477ae60c853)

The archive contains hundreds of thousands of files and having to listen to the HDD every time its scanned is very annoying and ultimately unnecessary.

Considering that digging up archived files, which are stored on large capacity HDD drives, is more likely than not to be a very common scenario, having the ability to set the update interval on paths which are not updated frequently (or at all) would probably be a worthwhile improvement.

**Comment by pantantrollo** [comment]
Correct, there are path that would barely have movement enough to be updated every few minutes.

**Comment by csswright** [comment]
I have quite a few external HDDs and I'd love for fsearch to retain the results for these offline disks too, similar to an offline indexer. I often want to search for files against both my local filesystem and the rest of my offline HDDs. I imagine fsearch can differentiate between online and offline files with visually in the results, like with offline entries being italicized or slightly grayed out. An update interval of "manual" or "on mount" would be super convenient.

Currently my workaround is to: 1) reproduce the tree hierarchy of my external drive as a empty files on the local host before it is detached, e.g. `~/files/my_external_hdd/cat.mp4` as an empty file which gets added to fsearch file database. 2) save a `tree` output of the list of files and their metadata like size and time as ~/log/index_my_hexternal_hdd.index`. Finding a file (alongside my local filesystem) is done with fsearch and then getting its metadata involves grepping for it in the file, an annoying two-step process.

**Comment by cboxdoerfer** [comment]
Hi, this is now possible on master and will be part of the upcoming 0.3 release.
